test/test_command.rb

```diff
@@ -12,7 +12,7 @@ class TestCommand < JekyllUnitTest
     context "when fatal error occurs" do
       should "exit with non-zero error code" do
         site = Object.new
-        stub(site).process { raise Jekyll::Errors::FatalException }
+        def site.process; raise Jekyll::Errors::FatalException; end
```

Comment:
actually that method will be only defined to that instance. `site` is not shared with other tests so it is fine.